pi-apps-terminal-bash-edition.sh

```diff
@@ -277,10 +274,9 @@ while [ "$1" != "" ]; do
 			exit 0
 		;;
 		*)
-			error "Unknown option '${light_blue}$1${red}'! run ${normal}${white}${dark_grey_background}pi-apps help${normal}${white}${red} to see all options."
+			error "Unknown option '${light_blue}$1${light_red}'! run ${normal}${white}${dark_grey_background}pi-apps help${normal}${white}${light_red} to see all options."
 		;;
 	esac
     shift
 done
-$PI_APPS_DIR/gui
```

Comment:
Why remove this? Without it pi-apps won't start normally if no arguments are provided.

Reply:
Fixed, thanks for informing.